src/main/java/language/LanguageCommand.java

```diff
@@ -0,0 +1,383 @@
+                                )
+                ))
+                .collect(Collectors.joining("\n"));
+    }
+
+    public static void setup(Velen velen, LanguageLogic languageLogic) {
+        VelenCommand.ofSlash(
+                "language",
+                "Query, Create, Connect, Remove Facets Languages",
+                velen,
+                new LanguageCommand(languageLogic),
+                new SlashCommandOptionBuilder()
+                        .setType(SlashCommandOptionType.SUB_COMMAND)
+                        .setName("add")
+                        .setDescription("Add a New Language")
+                        .setOptions(Lists.newArrayList(
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.STRING,
+                                        "name",
+                                        "Name",
+                                        true
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.STRING,
+                                        "description",
+                                        "Description"
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.BOOLEAN,
+                                        "constellation",
+                                        "Constellation"
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.BOOLEAN,
+                                        "court",
+                                        "Court"
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.BOOLEAN,
+                                        "Family",
+                                        "Family"
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.BOOLEAN,
+                                        "regional",
+                                        "Regional"
+                                ),
+                                SlashCommandOption.create(
+                                        SlashCommandOptionType.BOOLEAN,
+                                        "vulgar",
+                                        "Vulgar"
+                                )
+                        )),
+                new SlashCommandOptionBuilder()
+                        .setType(SlashCommandOptionType.SUB_COMMAND)
+                        .setName("remove")
+                        .setDescription("Remove Language")
+                        .setOptions(Lists.newArrayList(
+                                SlashCommandOption.create(
```

Comment:
Add autocomplete for language choice parameters

Reply:
For autocomplete, you have to enable the autocomplete option in your option, and then register a listener that takes the options and provides the user with a list of choices

Reply:
Yes that is one of the Draft Bits left